### ISSUE TITLE
Theme Customiser page fails with error 500

### ISSUE THREAD

**Issue by marcozambi** [presscustomizr/hueman#297]
Hello,
I have just upgraded Hueman to version 3.2.0 on WP 4.6.1.
Everything seems to work fine, with the very exception of customise.php.

Every time I try to open that page, this error appears in error.log of apache2
`PHP Fatal error:  require_once(): Failed opening required '/mypathto/mywebsite.com/wp-content/themes/hueman/functions/customizer/class-admin-customize.php' (include_path='.:/usr/share/php:/usr/share/pear) in /mypathto/mywebsite.com/wp-includes/template.php on line 572`

Just to clarify: 
- I have no plugins installed
- I am running a vanilla wp 4.6.1 upgraded from wp 4.5.2, no tampered code
- I have installed hueman 3.2.0 after having removed hueman 3.1.2
- I have no child theme

I checked the path `wp-content/themes/hueman/functions/customizer/class-admin-customize.php` and actually the file class-admin-customize.php is not there.
Could it be I have some "hooks" or setting still saved in WP DB from previous versions of Hueman?

**Comment by marcozambi** [comment]
Just to add that I did some further test, manually reinstalling old versions of Hueman.
- 3.1.2 ok
- 3.1.3 ok
- 3.1.4 custom.php page crashes with error 500

With 3.1.4 my error.log for apache2 returns this error
`PHP Fatal error:  require_once(): Failed opening required '/path_to_my_website/wp-content/themes/hueman/functions/customizer/class-admin-customize.php' (include_path='.:/usr/share/php:/usr/share/pear') in /path_to_my_website/wp-includes/template.php on line 572

**Comment by eri-trabiccolo** [comment]
Hello,
in 3.1.3 that file existed and was required from here:
https://github.com/presscustomizr/hueman/blob/v3.1.3/functions/init-core.php#L1057

Since 3.1.4 that file doesn't exist anymore, nor the reference to it (=> cannot find any attempt to load it in the theme's code).

All the relevant main (business logic) files are loaded from this file, even in the latest version;
https://github.com/presscustomizr/hueman/blob/v3.2.0/functions/init-core.php

Where there's no trace of any reference to "functions/customizer/class-admin-customize.php"

If that file in your install has it, this means that that file is not up to date :(

Hope this helps

**Comment by marcozambi** [comment]
Thanks for your answer. 
After hours of investigation I still cannot understand exactly what happens here. Let me explain a bit the scenario and what I found out up to now.

WHAT I NEED TO DO
I have a bunch of websites using WP and Human. WP has been successfully updated to 4.6.1, while Hueman is still at version 3.1.2, and of course I'd like to upgrade to the latest version (3.2.0).

WHAT HAPPENS
When I update Human from 3.1.2 to 3.2.0 via WP theme management interface, the update itself is confirmed as good and performed flawlessly. *All* functionalities of Hueman 3.2.0 appear to be up and running with the exception of: 
- Live preview
- Customise.php

SCENARIO I TRIED
First of all, let me specify I have no caching mechanisms in place whatsoever.
I've tried both a theme upgrade via WP interface (3.1.2->3.2.0 and 3.1.3->3.2.0), and also manually deleting the old Hueman directory in the file system (wp-content/themes/hueman) then expanding the zip file to form again the Hueman install dir.
Always the same result. 
At a culprit in my frustration, I also tried a more radical approach. I deleted *everything*, keeping only my data in the DB and the attachments folder. Everything else (so all WP, plugins, themes, everything) was gone. Downloaded and installed both WP 4.6.1 and Hueman 3.1.2, and then with basically a vanilla WP and Hueman source files, I updated the theme via WP interface from 3.1.2 to 3.2.0.
No joy, customiser.php still not working, failing error 500

WHAT I THINK
Having tried all aformentioned, I suspect that there is something, some config stuff or some hook or whatever saved in the db data, that for some reason is not removed jumping straight from 3.1.2 to 3.2.0
Possibly something was done for the 3.1.3->3.1.4 upgrade, maybe some code was cleaning up stuff. I have to expand all and read the code to understand what happens, but since I believe I am not the only one who might have to upgrade an old version of Hueman, it would be nice to have some hints by the developers about 
a) if indeed there is something that must be re-set o cleaned-up in the DB between different releases of Hueman and
b) in case, how to perform this cleanup manually
c) or I am completely wrong and there is something else in the picture I cannot understand yet.

Thanks in advance to all who will help :)
